index.html

```diff
@@ -116,7 +116,6 @@ <h2 class="project-title">Tonic</h2>
     </div>
 
```

Comment:
- [ ] It would be better to add a nav menu as we can see on the Figma design, kindly add it :recycle: 
- [ ] :thinking:  Kindly make sure you have a background image on your desktop version too in order to stick to the Figma design
- [ ] I think it would be better to reduce the distance between your social icons in order to stick to the Figma design.
- Your design:
![Screenshot from 2023-02-23 12-01-00](https://user-images.githubusercontent.com/97834160/220876234-6aa05156-47b5-459c-aa56-cc4bebb77403.png)
- Figma:
![Screenshot from 2023-02-23 12-17-26](https://user-images.githubusercontent.com/97834160/220879796-e6cd17af-9d22-4bd3-a02d-6b04763ea767.png)

Reply:
@GedeonTS Thank you for taking the time to give me those reviews. I will take it into account. But for the background image, I added it and it's displaying on my side either on the mobile or the desktop version.

Reply:
@GedeonTS  Please are you there?

Reply:
![Capture_back](https://user-images.githubusercontent.com/119133987/220882999-069817de-c86d-4679-85f8-b4b97819a169.JPG)

Reply:
Hi again @adeola003, for the background Image you can verify if the path is set properly as I used raw.githack to test the project,

Reply:
Thank you very much @GedeonTS